lib/findingaids/ead/behaviors.rb

```diff
@@ -94,6 +94,17 @@ def get_ead_creators
     get_ead_creators.map(&:text).flatten.compact.uniq.sort
   end
 
+  #getting places and scrubbing out subfield demarcators
+  def get_ead_places
+    gep = search("//geogname")
+    get_ead_places = []
+    gep.children.each{|n|
+        t = n.text
+        get_ead_places.push(fix_subfield_demarcators(t))
+    } 
+    get_ead_places.compact.uniq.sort  #don't need to flatten since already an array
+  end
```

Comment:
@eshadatta With ruby array functions you can pare this down:

```ruby
def get_ead_places
   @get_ead_places ||= search("//geogname").map {|field| fix_subfield_demarcators(field.text) }.compact.uniq.sort
end
```

Then you can add the following to the `ead_template` and the `ead_component`:

<geogname source="lcsh">Spain |x History |y Civil War, 1936-1939 |x Atrocities.</geogname>